boot/flags.go

```diff
@@ -280,3 +282,60 @@ func setNextBootFlags(dev Device, rootDir string, flags []string) error {
 
 	return m.Write()
 }
+
+// RunModeRootfs returns where the run mode root filesystem is mounted for the
+// given mode.
+// For run mode, it's just "/run/mnt/data".
+// For install mode it's "/run/mnt/ubuntu-data".
+// For recover mode it's either "/host/ubuntu-data" or "" if that is not
+// mounted. Note that, for recover  mode, this function only returns a non-empty
+// return value if the partition is mounted and trusted, there are certain
+// corner-cases where snap-bootstrap in the initramfs may have mounted
+// ubuntu-data in an untrusted manner, but for the purposes of this function
+// that is ignored.
+// This is primarily meant to be consumed by "snap{,ctl} system-mode".
+func RunModeRootfs(mode string) (string, error) {
+	runDataRootfs := ""
+	switch mode {
+	case ModeRun:
+		runDataRootfs = InitramfsDataDir
```

Comment:
I wonder if this function or some other way we should make it explicit that data in run mode is also available at / . OTOH maybe it's known implicitly well enough?

Reply:
I suppose this could return a list, and for run mode we could have `["/", "/run/mnt/data"]` ?

Reply:
This is done